subsys/bluetooth/shell/bt.c

```diff
@@ -1154,6 +1160,32 @@ static int cmd_clear(int argc, char *argv[])
 
 	return 0;
 }
+
+static int cmd_ch_map(int argc, char *argv[])
+{
+	u8_t channel_map[5];
+	int err;
+
+	if (argc != 2)
+	{
+		return -EINVAL;
+	}
+
+	err = hexstr2array(argv[1], channel_map, 5);
+	if (err) {
+		printk("Invalid channel map\n");
+		return err;
+	}
+
+	err = bt_le_set_ch_map(channel_map);
```

Comment:
Let's use `chan` everywhere. Not `ch`, not `channel`. We already do this in other parts of the code.

Reply:
Good point, I will do that.